src/build-time-render/BuildTimeRender.ts

```diff
@@ -113,6 +113,18 @@ class MockCompilation {
 	}
 }
 
+const relRegExp = /rel\=\"([\w]+)\"/;
```

Comment:
Could this be changed to be slightly simpler?

```ts
const.dynamicLinkRegExp = /rel\=(\"|\')(preconnect|prefetch|preload|prerender|dns-prefetch|stylesheet)(\"|\')/;
```

and then used like `dynamicLinkRegExp.test(linkString)` in the places? 

Also as it stands the regex actually wouldn't work for `dns-prefetch`